streamrip/client/downloadable.py

```diff
@@ -101,15 +101,13 @@ def __init__(self, session: aiohttp.ClientSession, info: dict):
         self.id = str(info["id"])
 
     async def _download(self, path: str, callback):
-        # with requests.Session().get(self.url, allow_redirects=True) as resp:
         async with self.session.get(self.url, allow_redirects=True) as resp:
             resp.raise_for_status()
             self._size = int(resp.headers.get("Content-Length", 0))
             if self._size < 20000 and not self.url.endswith(".jpg"):
                 try:
                     info = await resp.json()
                     try:
-                        # Usually happens with deezloader downloads
                         raise NonStreamableError(f"{info['error']} - {info['message']}")
                     except KeyError:
                         raise NonStreamableError(info)
```

Comment:
Also, is this error handling still necessary? If not, I can remove it as well and just `raise NonStreamableError(info)`